app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -113,6 +118,7 @@ object ApiConnector {
     case object GetEditSetRecord extends SID("OSGESR001")
     case object UpdateEditSetRecord extends SID("OSUESR001")
     // TODO: The real SID will be provided by Adam once he figures out the schema.
+    case object GetLegalStatusSummary extends SID("OSLISALS001")
```

Comment:
I confirmed that this ID is correct.

However, if only for consistency, I'd suggest to rename it to `GetLegalStatuses`.

Reply:
Renamed. Would be helpful to match the name to the API service name (in addition to the service identifier) but we can discuss this as a team once the Async API services are fully defined.